CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# logrotate contributions
+
+## Pull requests
+
+  - Fork it.
+  - Create your feature branch (`git checkout -b fixing-blah`), please avoid working directly on the `master` branch.
+  - Check for unnecessary whitespace with `git diff --check` before committing.
+  - Add your change to `ChangeLog.md` under the `UNRELEASED` section
```

Comment:
I would prefer to add only **user visible changes** to ChangeLog.md.  Implementation details can be easily found in git history.